php/src/Communicator.cpp

```diff
@@ -2094,15 +2053,11 @@ IcePHP::ValueFactoryManager::find(string_view id) const noexcept
 
     CommunicatorInfoIPtr info = p->second;
 
-    Ice::ValueFactoryPtr factory;
-    if (id.empty())
+    ValueFactoryPtr factory = info->findFactory(id);
+    if (!factory && id.empty())
```

Comment:
we are missing a `std::move` call in the factory capture below.

Reply:
Fixed